cmd/carbonapi/main.go

```diff
@@ -38,5 +41,14 @@ func main() {
 		)
 	}
 	fh.Close()
-	capi.StartCarbonapi(api, logger)
+	expvar.NewString("BuildVersion").Set(BuildVersion)
+	logger.Info("starting carbonapi",
+		zap.String("build_version", BuildVersion),
+		zap.Any("apiConfig", api),
+	)
+	app, err := capi.InitializeApp(api, logger, BuildVersion)
```

Comment:
matter of taste: personally I'd probably call this just "New", "Initialize" to me sounds like it's taking an empty struct and populating the fields, whereas this builds a new struct and returns it.